### ISSUE TITLE
Close Magik Connection should not call exit()

### ISSUE THREAD

**Issue by lczub** [lczub/robotframework-magik#12]
To avoid communication timing problems at the end of a test run against Smallworld 5 images, an additional sleep before closing the connection helps. Example

```
Close Magik Connection
    [Documentation]    Closes the telnet connection to remote_cli of a Smallworld Magik image
    ...
    ...    Returns any remaining output.
    Write Bare    exit()\n
    Sleep    10ms
    ${out}=    Close Connection
    [Return]    ${out}

```

**Comment by lczub** [comment]
Keyword _Close Magik Connection_ has called _exit()_ before closing the connecting.
  This was not correct, cause _exit()_ is only known inside the SW GIS internal cli client. 
  Under GIS 4.0, this failure has no negativ effect. But under GIS 5.0, the remote_cli tries to 
  write a traceback to the closed streams. This create than an endless traceback loop.

**Comment by lczub** [comment]
fixed with v0.4.0-Alpha
